challenge.js

```diff
@@ -57,5 +60,8 @@
  */
 
 // ANSWER
+var parent;
+var p = document.createElement("p");
+parent.appendChild(p);
```

Comment:
Given a reference to an HTML element attached to the DOM stored in `parent`, this works flawlessly. We didn't cover `Node.appendChild` in the lesson, so it's clear you successfully looked it up. 1